.github/workflows/ci.yml

```
@@ -0,0 +1,104 @@
+        uses: actions/cache@v4
+        with:
+          path: |
+            ~/.bun/install/cache
+            node_modules
+          key: ${{ runner.os }}-bun-${{ hashFiles('**/bun.lockb') }}
+          restore-keys: |
+            ${{ runner.os }}-bun-
+
+      # Install security tools
+      - name: Install security tools
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:5:56: Quote this to prevent word splitting

(shellcheck)